README.md

```diff
@@ -1,8 +1,6 @@
-# TutorPoint Development Branch
+# TutorPoint Master Branch
 Repository for Group 2, MEng Software Engineering Project.
 
-The development branch is seperate to the master in that the master must always be fully functioning, customer presentable version of the software.
```

Comment:
Maybe change this back to the dev branch one, or since this is always causing issues we could just have a generic tutorpoint readme that all the branches use after we finish this merge?

Reply:
README.md was never designed to have different files for different branches - it's intended to be a generic readme for the whole project. Because of this, git treats it the same as any other version controlled file, so its almost impossible to automate it not getting merged. So yeah I'd definitely suggest just having one readme for all the branches.